payments_service/braintree/templates/braintree/emails/subscription_canceled.html

```diff
@@ -9,7 +9,7 @@
         </tr>
         <tr class="desc">
             <th scope="row">{{ product.description }}</th>
-            <td>${{ moz_trans.amount }} <span>per month</span></td>
+            <td>{{ recurring_amount }}</td>
```

Comment:
`amount_and_terms` ? I dunno, 'recurring' won't always be true so this seems odd. But, meh, names.